.github/workflows/nodejs.yml

```diff
@@ -39,11 +39,15 @@ jobs:
       env:
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - name: Current directory
+      run: |
+        pwd
+        ls -al
     - name: Code Coverage Report
       uses: codecov/codecov-action@v1
       with:
         token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
-        file: ./coverage/coverage-final.json # optional
+        file: ./coverag*/coverage-final.json # optional
```

Comment:
@BKristenssonAlfsson is the name change from coverage to coverag* intentional? Not sure if it makes a big difference, just curious.

Reply:
I added the asterix as I separated the test folders to find both with same line